tests/test_conversions_tools.py

```diff
@@ -0,0 +1,49 @@
+import pathlib
+import unittest
+
+import numpy as np
+
+from lsst.cst.conversions import ids_to_str, nearest_patch_from_ra_dec
+
+PATH = pathlib.Path(__file__).parent.absolute()
+
+
+class TestDataUtils(unittest.TestCase):
```

Comment:
Test should be disabled because TAPService used inside nearest_patch_from_ra_dec is not available on test environment. Alternative is to create a test notebook that can be added to the launching tests.

->    @unittest.skip("Temporarily unused until a way to test is found")

Reply:
Ah! OK -- I thought it was being skipped because it was unclear how to test it. I added some assert statements. But I added back the line to make it skip the test entirely, since the problem is the reliance on TAP:

`@unittest.skip("Disabled because TAP Service is not available in test environment.")`